src/style.css

```diff
@@ -0,0 +1,83 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.form {
+  display: flex;
+  width: 100%;
+  height: 70px;
+  margin-top: 20px;
+}
+
+.form-small {
+  height: 42px;
+}
+
+.label {
+  display: block;
+  position: relative;
+  width: 100%;
+}
+
+.label::before {
+  position: absolute;
+  content: "";
+  height: 19px;
+  width: 19px;
+  top: 25px;
+  left: 26px;
+  background-image: url(images/Search.svg);
+  background-size: contain;
+  background-repeat: no-repeat;
+  z-index: 1;
+}
+
+.label-small::before {
+  height: 11px;
+  width: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.search {
+  width: 100%;
+  height: 100%;
+  padding-left: 62px;
+  font-family: inherit;
+  font-size: 16px;
+  font-weight: 400;
+  color: #3d4e61;
```

Comment:
It is a good practice to make all colors [variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)